g3doc/obtaining-training-data.md

```diff
@@ -0,0 +1,50 @@
+# Obtaining Training Data
+
+This document gives an overview of how to use the Gematria tooling to
+parse/process training data from multiple sources.
+
+## Parsing training data from the BHive dataset
+
+The publically available [BHive](https://github.com/ithemal/bhive) dataset
+has a decent amount of training data available in CSV format. Gematria
```

Comment:
I'd replace "a decent" with "~300k basic blocks and their throughput for multiple microarchitectures".